main.go

```diff
@@ -48,23 +47,19 @@ func main() {
 	//service healthcheck
 	svc.Router().Path("/healthcheck").Methods("GET").HandlerFunc(handlers.HealthCheck)
 
-	//connect to cache-broker
-	log.Info("Getting cache-broker", log.Data{"cache-broker": cacheBroker})
-	publisher := broker.NewBroker() //incoming messages
-
 	//Streaming Request Handler
 	streamHandler := handlers.Streaming{
 		RequestTimeout:    time.Duration(cfg.RequestTimeout),
 		HeartbeatInterval: time.Duration(cfg.HeartbeatInterval),
-		Broker:            publisher,
+		Logger:            logger.NewLogger(),
 	}
 
-	streamHandler.AddStream(svc.Router(), "/filings", filingHistoryStream)
-	streamHandler.AddStream(svc.Router(), "/companies", companyProfileStream)
-	streamHandler.AddStream(svc.Router(), "/insolvency-cases", companyInsolvencyStream)
-	streamHandler.AddStream(svc.Router(), "/charges", companyChargesStream)
-	streamHandler.AddStream(svc.Router(), "/officers", companyOfficersStream)
-	streamHandler.AddStream(svc.Router(), "/persons-with-significant-control", companyPSCStream)
+	streamHandler.AddStream(svc.Router(), "/filings", filingHistoryStream, cfg.CacheBrokerURL)
```

Comment:
The cache broker URL should be a field-level property, not a method parameter as it won't change.